src/ray/raylet/local_task_manager.cc

```diff
@@ -750,21 +751,37 @@ bool LocalTaskManager::PinTaskArgsIfMemoryAvailable(const TaskSpecification &spe
       }
     }
   }
-
   *args_missing = false;
+
+  // Pinning bookkeeping
```

Comment:
this comment is not useful. either delete it or make it self-explanatory, for example:
```suggestion
  // Update fields tracking pinned task arguments.
```